src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/InProcess/EditorInProcess_Classification.cs

```diff
@@ -94,16 +94,22 @@ public async Task VerifyGetClassificationsAsync(IEnumerable<ClassificationSpan>
         var actualClassifications = await GetClassificationsAsync(cancellationToken);
         var actualArray = actualClassifications.ToArray();
         var expectedArray = expectedClassifications.ToArray();
+        var actualOffset = 0;
 
         for (var i = 0; i < actualArray.Length; i++)
         {
-            var actualClassification = actualArray[i];
+            var actualClassification = actualArray[i+actualOffset];
             var expectedClassification = expectedArray[i];
 
+            if (actualClassification.ClassificationType.BaseTypes.Count() == 1 && actualClassification.ClassificationType.BaseTypes.Any(t => t is ILayeredClassificationType layered && layered.Layer == ClassificationLayer.Syntactic))
```

Comment:
Can simply the pattern a little, since `layered` is not used outside the condition:
```suggestion
            if (actualClassification.ClassificationType.BaseTypes.Count() == 1 && actualClassification.ClassificationType.BaseTypes.Any(t => t is ILayeredClassificationType { Layer: ClassificationLayer.Syntactic }))
```

I hate that all of these properties return just `IEnumerable` though, so we can't use list patterns, which would make it even simpler